.buildkite/scripts/lifecycle/pre_command.sh

```diff
@@ -26,8 +26,17 @@ retry 5 15 install_deps
 
 cd ..
 
+echo '--- Agent Debug/SSH Info'
 node .buildkite/scripts/lifecycle/print_agent_links.js || true
 
+if [[ "$(curl -is metadata.google.internal || true)" ]]; then
+  echo ""
+  echo "To SSH into this agent, run:"
+  echo "gcloud compute ssh --tunnel-through-iap --project elastic-kibana-ci --zone \"$(curl -sH Metadata-Flavor:Google http://metadata.google.internal/computeMetadata/v1/instance/zone)\" \"$(curl -sH Metadata-Flavor:Google http://metadata.google.internal/computeMetadata/v1/instance/name)\""
```

Comment:
Do we have any instructions to keep the agent active?

Reply:
We'll have to build a bunch of stuff to make that happen... can we track that separately?

Reply:
(Right now, you would just have to put a `sleep` somewhere)